cherrypy/process/servers.py

```diff
@@ -237,8 +243,22 @@ def wait(self):
             # Wait for port to be occupied if not running via socket-activation
             # (for socket-activation the port will be managed by systemd )
             if isinstance(self.bind_addr, tuple):
-                host, port = self.bind_addr
-                wait_for_occupied_port(host, port)
+                host, port = self.bound_addr
```

Comment:
Aha. 'host' is also unused. It was the `host, port = self.bound_addr` which is unused and needs to be removed.